ext/drreg/drreg.h

```diff
@@ -75,6 +75,18 @@ typedef enum {
  * INIT
  */
 
+/**
+ * Register classes used to denote which types of registers
+ * to spill or restore.
+ */
+typedef enum {
+    DRREG_INVALID_SPILL_CLASS,  /* denotes invalid class */
```

Comment:
These all need to be Doxygen comments, and properly capitalized + punctuated.

Reply:
Done.